### DISCUSSION TITLE
Era support for zkExport tool

### DISCUSSION THREAD

**Discussion by AnimYogi** [JackHamer09/zkExport#2] [Q&A] [answered]
Hello, I was wondering if is there any way to export era transactions? Thank you!

**Comment by JackHamer09** [comment]
Unfortunately not at the moment :(
I plan to add support for era transactions once the official API will be released (that will be in the next couple of months)

**Reply by JackHamer09** [reply, marked as answer]
zkExport tool for zkSync Era is now available on - https://zkexport.netlify.app 🚀
